simulation/executor/types.go

```diff
@@ -0,0 +1,175 @@
+package simulation
```

Comment:
unsure how you feel about bringing this into osmosis @ValarDragon . I felt like it would potentially be helpful to have direct control over the OperationMsg struct specifically